data/json/monsters/reptile_amphibian.json

```
@@ -35,6 +35,7 @@
     "armor_bash": 4,
     "harvest": "mutant_animal_large_noskin",
     "path_settings": { "max_dist": 5 },
+    "upgrades": { "half_life": 20, "into": "mon_frog_mother" },
```

```suggestion
    "upgrades": { "half_life": 50, "into": "mon_frog_mother" },
```
The proposed monster is an absolute beast, probably best we delay them a bit